src/Utils.js

```diff
@@ -35,33 +17,14 @@ export function isCustomComponentElement(inst) {
   return !!inst && React.isValidElement(inst) && typeof inst.type === 'function';
 }
 
-export function propsOfNode(node) {
-  if (REACT013 && node && node._store) {
-    return (node._store.props) || {};
-  }
-  if (node && node._reactInternalComponent && node._reactInternalComponent._currentElement) {
-    return (node._reactInternalComponent._currentElement.props) || {};
-  }
-  if (node && node._currentElement) {
-    return (node._currentElement.props) || {};
-  }
-  if (REACT15 && node) {
-    if (internalInstance(node) && internalInstance(node)._currentElement) {
-      return (internalInstance(node)._currentElement.props) || {};
-    }
-  }
-
+function propsOfNode(node) {
```

Comment:
perhaps, yes. I've been trying to separate things mentally between adapters and the core enzyme lib so that they are easy to separate, and it's easier for me to find when things are no longer used